src/core/plugins/spec/actions.js

```diff
@@ -100,7 +100,7 @@ export const resolveSpec = (json, url) => ({specActions, specSelectors, errActio
     url = specSelectors.url()
   }
 
-  let { getLineNumberForPath } = AST
+  let getLineNumberForPath = AST.getLineNumberForPath ? AST.getLineNumberForPath : () => undefined
```

Comment:
wasn't the entire AST code nuked? 😕

Reply:
I guess not https://github.com/swagger-api/swagger-ui/commit/0359f9c3649072dffdad6a39ab08a9eb60647afa